state/test/test_state_proof_verification.py

```diff
@@ -75,7 +76,9 @@ def add_prefix_nodes_and_verify(state, prefix, keys_suffices=None):
     for k, v in key_vals.items():
         state.set(k, v)
 
-    prefix_prf = state.generate_state_proof_for_key_prfx(prefix.encode())
+    prefix_prf, val = state.generate_state_proof_for_keys_with_prefix(prefix.encode(), get_value=True)
```

Comment:
Do we have a test where `get_value=False`?

Reply:
Added